image-customize

```diff
@@ -144,6 +116,39 @@ def install_packages(machine_instance, packagelist, install_command):
         machine_instance.execute(install_command + " " + ' '.join(allpackages), timeout=1800)
 
 
+#
```

Comment:
I rather enjoy defining a `main()` function and adding at the very end:

```
if __name__ == '__main__':
    main()
```

Reply:
Sure! I'll let the first wave of tests run, and then change this.

Reply:
Done. This spotted another little issue.